libraries/rush-lib/src/cli/actions/InstallAction.ts

```diff
@@ -46,6 +47,9 @@ export class InstallAction extends BaseInstallAction {
 
   protected async buildInstallOptionsAsync(): Promise<IInstallManagerOptions> {
     const terminal: Terminal = new Terminal(new ConsoleTerminalProvider());
```

Comment:
Same feedback here-- can we use `this._terminal` from the base class instead?

Reply:
changed